butterfree/pipelines/feature_set_pipeline.py

```diff
@@ -16,6 +18,8 @@ class FeatureSetPipeline:
         feature_set: feature set composed by features and context metadata.
         sink: sink used to write the output dataframe in the desired locations.
         spark_client: client used to access Spark connection.
+        timezone: timestamp feature transformations will assume this timezone 
+            when they don't have a tz suffix.
```

Comment:
I'd just inform here that spark con config (spark.sql.session.timeZone) and an env variable (TZ) will be set with this value.